src/HttpHeader.cc

```diff
@@ -349,6 +351,10 @@ HttpHeader::Isolate(const char **parse_start, size_t l, const char **blk_start,
     return end;
 }
 
+/// Locates and Parses headers stored in a buffer.
+/// \retval 1   and sets hdr_sz on success
+/// \retval 0   when needs more data, sets hdr_sz to 0
+/// \retval -1  on error, sets hdr_sz to 0
 int
```

Comment:
could we use an enum class instead of an int to signal status?

Reply:
Not sure yet. The main objective though is to remove entirely. This kind of legacy API I'd rather just leave as-is unless there is a strong reason to do changes before the removal can happen.